lib/util/prompt-suggestion.js

```diff
@@ -115,7 +115,15 @@ promptSuggestion.prefillQuestions = (store, questions) => {
 
     const storedValue = promptValues[question.name];
 
-    if (storedValue === undefined) {
+    if (
+      (storedValue === undefined) ||
+      (
+        Object.prototype.hasOwnProperty.call(question, 'choices') &&
```

Comment:
Do we need this line? if `question.choices === undefined`, the `isFunction` check will still returns false.

Reply:
Good catch. Removed.